### ISSUE TITLE
[Feature request] Remove rounded corners

### ISSUE THREAD

**Issue by zapashcanon** [micheleg/dash-to-dock#378]
Hi,

(I know it has already been asked in #75, but this one is a little bit different as it's not about removing them when the dock is extended, but when it's not.)

It would be great to add an option to disable the dash rounded corners, so it could look like this:

![capture du 2016-09-17 19-19-20-small](https://cloud.githubusercontent.com/assets/16472988/18609845/02fc78c6-7d0c-11e6-9c5f-8d7ea51c01e5.png)

You would have to add the following to `schemas/org.gnome.shell.extensions.dash-to-dock.gschema.xml`:

``` xml
<key type="b" name="remove-rounded-corners">
    <default>false</default>
    <summary>Remove rounded corners</summary>
</key>
```

It even could be enabled by default, as it looks better - in my opinon at least, I don't know what's the main preference.

Enabling the option would change `stylesheet.css` like this:

``` css
[...]

#dashtodockContainer.shrink.left #dash,
#dashtodockContainer.dashtodock.left #dash {
    border-left: 0px;
    border-radius: 0px 0px 0px 0px;
}

#dashtodockContainer.shrink.right #dash,
#dashtodockContainer.dashtodock.right #dash {
    border-right: 0px;
    border-radius: 0px 0px 0px 0px;
}

#dashtodockContainer.shrink.top #dash,
#dashtodockContainer.dashtodock.top #dash {
    border-top: 0px;
    border-radius: 0px 0px 0px 0px;
}

#dashtodockContainer.shrink.bottom #dash,
#dashtodockContainer.dashtodock.bottom #dash {
    border-bottom: 0px;
    border-radius: 0px 0px 0px 0px;
}

[...]
```

But I don't know the codebase well enough to do the other changes...

Moreover, the same option could be provided for these corners:

![capture du 2016-09-17 19-19-20-verysmall](https://cloud.githubusercontent.com/assets/16472988/18609906/eed4cb30-7d0d-11e6-83ee-9abd1bd921bb.png)

It should work by doing almost the same changes I guess.

**Comment by gayanper** [comment]
I have already done some changes at https://github.com/gayanper/dash-to-dock/tree/rnd-borders. If there has not been any work started and my changes are ok, i can make a pull request.

**Comment by zapashcanon** [comment]
@micheleg, this can be closed thanks to #409.
